battleship-client/src/main/java/com/epam/solid/nie/client/ui/StartSceneController.java

```diff
@@ -10,28 +10,26 @@
 import javafx.scene.control.TextField;
 
 import java.io.IOException;
-import java.util.logging.Logger;
 
 public class StartSceneController {
-    private Logger logger = Logger.getLogger(StartSceneController.class.getName());
 
     @FXML
-    private Button btnConnect;
+    private Button btn_connect;
```

Comment:
I get the impression that this underscore separator breaks the naming convention of your code